crypto/err/err.c

```diff
@@ -859,46 +731,40 @@ void ERR_remove_state(unsigned long pid)
 }
 #endif
 
+static void err_do_clean(void *p)
```

Comment:
I'm not sure this is necessary. The auto-deinit code will automatically call ERR_remove_thread_state() on thread stop - and that should work on Windows too whereas this only works on pthreads IIRC.